Assets/Scripts/Gameplay/DebugCheats/DebugCheatsManager.cs

```diff
@@ -130,24 +130,24 @@ public void GoToPostGame()
             GoToPostGameServerRpc();
         }
 
-        [ServerRpc(RequireOwnership = false)]
-        void SpawnEnemyServerRpc(ServerRpcParams serverRpcParams = default)
+        [Rpc(SendTo.Server, RequireOwnership = false)]
+        void SpawnEnemyServerRpc(RpcParams serverRpcParams = default)
```

Comment:
You will want to change all of the methods to just use the Rpc suffix and exclude the legacy ClientRpc and ServerRpc suffixes. 

_(Apply to all ClientRpc and ServerRpc methods adjusted)_

Reply:
+1 to this, it also helps with the clarity. It's faster to look at a prefix vs something in the middle of the world, to udnerstand where something is being called from (especially if you're not in the same class and can't see the attributes)

Reply:
Done!